readme.md

```diff
@@ -230,6 +247,13 @@ queryString.stringify({foo: [1, 2, 3]});
 //=> 'foo=1&foo=2&foo=3'
 ```
 
+##### arrayFormatSeparator
+
+Type: `string`\
+Default: `','`
+
+Character for arrayFormat when `separator` behavior chosen.
```

Comment:
I think this could be written more clearly.
```suggestion
The character used to separate array elements when using `{arrayFormat: 'separator'}`.
```